### DISCUSSION TITLE
can ruffle load data URIs?

### DISCUSSION THREAD

**Discussion by Hellscaped** [ruffle-rs/ruffle#9291] [Q&A] [answered]
I'm trying to make a py script to encode swfs and make swf -> html, but ruffle refuses to load the swf for some reason.

**Comment by Hellscaped** [comment, marked as answer]
I got it working! Ruffle on local files!

**Reply by n0samu** [reply]
I'm curious to hear how you did it. I didn't think that would be possible.

**Reply by Hellscaped** [reply]
suprisingly I was able to just use player.load()
